src/dataflow/src/sink/kafka.rs

```diff
@@ -24,6 +26,8 @@ use expr::GlobalId;
 use interchange::avro::{DiffPair, Encoder};
 use repr::{RelationDesc, Row};
 
+static KAFKA_SINK_FUEL: usize = 10000;
```

Comment:
Can we make this either be part of the `KafkaSinkConnector` or an argument to `kafka`? I figure it is a reasonable knob to want to dial around, and having it be hardwired seems like something we'll want to fix at some point.